sycl/include/CL/sycl/handler.hpp

```diff
@@ -895,16 +895,32 @@ class handler {
   }
 
   // Explicit copy operations API
+  template <access::target AccessTarget>
+  constexpr static bool isConstOrGlobal() {
```

Comment:
I thought that one of the advantages of constexpr functions is that we can avoid templates.
Please, make `AccessTarget` a function parameter rather than template parameter.